spec/migration_spec.rb

```diff
@@ -141,3 +141,21 @@
     end
   end
 end
+
+migration '240814115647-disable-reverse-proxy-by-default' do
```

Comment:
I swear I created a PR for it, but I only had a local branch. Now I opened https://github.com/theforeman/foreman-installer/pull/959 to split it out over multiple files. I'd prefer that over one huge file.

Reply:
Rebased on your merged split out